src/checker/tests.py

```diff
@@ -187,7 +187,8 @@ def test_script_timeout(self):
             self.solution.check_solution()
             for checkerresult in self.solution.checkerresult_set.all():
                 self.assertIn('1', checkerresult.log, "Test did not even start?")
-                self.assertIn('Timeout occured!', checkerresult.log, "Test result does not mention timeout")
+                self.assertIn('Timeout occurred!', checkerresult.log, "Test result does not mention timeout")
+                # TODO: 'occured vs occurred' ? (string defined in checker/basemodels.py : CheckerResult.set_log)
```

Comment:
```suggestion
```

The error message was adjusted accordingly in `checker/basemodels.py`. From what I can see, this TODO comment is obsolete. Or am I missing something?

Reply:
Yes, the TODO followed by the question could be removed. But the pointer where the string has been defined I would like to keep.

Reply:
changed via https://github.com/KITPraktomatTeam/Praktomat/pull/352/commits/5b0d3ef4f5430b6471c8d1b24a51a41515a3888f